README.md

```diff
@@ -63,31 +63,46 @@ noid_service.valid? 'xyz123foobar'
 
 ## ActiveFedora integration
 
-To get ActiveFedora to automatically call your Noid service whenever a new ActiveFedora object is saved, include the `ActiveFedora::Noid::Model`, e.g.:
+To get ActiveFedora to automatically call your Noid service whenever a new ActiveFedora object is saved, include the `Noid::Rails::Model`, e.g.:
 
 ```ruby
 # app/models/my_object.rb
-require 'active_fedora/noid'
+require 'noid-rails'
 
 class MyObject < ActiveFedora::Base
-  include ActiveFedora::Noid::Model
```

Comment:
It depends on ActiveFedora conventions, namely overriding `assign_id`.  It would be better for us to update ActiveFedora to call a configured service from within `assign_id` (and provide a NilService by default)

Reply:
Good call.